test/basic-querying.test.js

```diff
@@ -154,7 +159,7 @@ describe('basic-querying', function() {
       User.find(function(err, users) {
         should.exists(users);
         should.not.exists(err);
-        users.should.have.lengthOf(6);
+        if (!isCassandraConnector) users.should.have.lengthOf(6);
```

Comment:
So how many users are returned by the `find` method when using cassandra connector? Why is the number different from how (almost) every other connector returns?